main.py

```diff
@@ -799,7 +843,8 @@ def run_analysis(self) -> None:
             log(f"Target URL: {self.target_url}")
             print_line_separator()
 
-            check: bool = check_site_is_available(self.target_url)
+            check = True
+            # bool = check_site_is_available(self.target_url)
```

Comment:
Did you mean to leave this commented out? If we don't need the check we should just pull these lines out.

Reply:
Comented while testing to not save a bit of time on each run. back to normal.